config/base/common/namespace.yaml

```diff
@@ -3,6 +3,6 @@ kind: Namespace
 metadata:
   labels:
     app.kubernetes.io/part-of: olm
-    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce: restricted
```

Comment:
@joelanford ^ JFY

Reply:
@camilamacedo86 can you identify which, if any, current e2e test might be able to verify this doesn't cause a problem? Maybe just?:

in `./kind-config.yaml` Should we turn on PodSecurity?
```diff
    kubeadmConfigPatches:
      - |
        kind: ClusterConfiguration
        apiServer:
            extraArgs:
-              enable-admission-plugins: OwnerReferencesPermissionEnforcement
+              enable-admission-plugins: OwnerReferencesPermissionEnforcement,PodSecurity
```

Reply:
e2e seems to pass for me with this set, so maybe it's all good?

Reply:
just trying to verify it's really used if I set it there

Reply:
When we install OLMv1 the ns will be set within right
So, the tests running on that. But do you still want any extra step? if so, can you please clarify? Sorry, I could not follow what are you asking for I do in this case.

Reply:
nvm, seems like at our current version enforced for Kind, we've got PodSecurity by default. Verified it bounced a non-compliant pod trying to be created in an NS set restricted. Since our current e2e do deployment, they must be okay.